src/parser/jobs/sge/modules/Overheal.tsx

```diff
@@ -12,22 +12,22 @@ export class Overheal extends CoreOverheal {
 
 	override trackedHealCategories = [
 		{
-			name: <Trans id="sge.overheal.hot.name">HoTs</Trans>,
+			name: <Trans id="sge.overheal.addersgall.name">Addersgall</Trans>,
 			color: SuggestedColors[1],
 			trackedHealIds: [
-				this.data.statuses.PHYSIS.id,
-				this.data.statuses.PHYSIS_II.id,
 				this.data.statuses.KERAKEIA.id,
+				this.data.actions.DRUOCHOLE.id,
+				this.data.actions.IXOCHOLE.id,
+				this.data.actions.TAUROCHOLE.id,
 			],
 		},
 		{
 			name: <Trans id="sge.overheal.cooldowns.name">Cooldowns</Trans>,
 			color: SuggestedColors[2],
 			trackedHealIds: [
-				this.data.actions.DRUOCHOLE.id,
-				this.data.actions.IXOCHOLE.id,
-				this.data.actions.TAUROCHOLE.id,
 				this.data.actions.HOLOS.id,
+				this.data.statuses.PHYSIS.id,
```

Comment:
Probably don't need to include Physis here, since it upgrades directly to Physis II and shouldn't matter at level 60+. Looks fine thought, and this doesn't hurt anything, so I'm happy to merge it if you want to leave this as-is.

Reply:
Let's just leave it for now. I originally added it for completeness since I wasn't paying attention to what level things came at. Might drop it later since it won't be relevant in any Ultimate, but it's fine for now.